web/src/lib/components/asset-viewer/activity-status.svelte

```diff
@@ -29,8 +29,9 @@
       <Icon path={mdiCommentOutline} class="scale-x-[-1]" size={24} />
       {#if numberOfComments}
         <div class="text-xl">{numberOfComments}</div>
-      {:else if !isShowActivity}
-        <div class="text-lg">{$t('say_something')}</div>
+      <!-- the say_something text is removed for being obtrusive but could be put back in -->
+      <!-- if it disappears on narrowing the viewport or once user control auto fadeout is implemented -->
+      <!-- {:else if !isShowActivity} <div class="text-lg">{$t('say_something')}</div> -->
```

Comment:
How about only hiding it on mobile?

Reply:
Maybe the way forward is lots of screenshots. Fortunately, the usual-looking video player is back today on my setup lmao.

Reply:
First idea: disable the text whenever the info pane is showing (existing behaviour: only if activity pane is showing.
(I will always be pro–always disable personally because no other user buttons in the asset viewer have text).

**immich (release):**
![image](https://github.com/immich-app/immich/assets/28405293/9b14d26d-289b-4cea-9eff-393a9a51107c)

**immich (pr branch):**
![image](https://github.com/immich-app/immich/assets/28405293/c7ef05e4-9d3b-4d16-a053-be9101d84177)

**google photos:**
![image](https://github.com/immich-app/immich/assets/28405293/5ebfc6d9-199c-445d-a9f6-a983fada036a)

Reply:
Second idea: <600px viewport width (examples are 550×400px)

**immich (release):**
![image](https://github.com/immich-app/immich/assets/28405293/fdffde16-53d6-48b8-9970-54038ae699ea)

**immich (pr branch):**
![image](https://github.com/immich-app/immich/assets/28405293/ab46fb86-31e7-41ec-8d8c-0ed95e9e7e09)

**google photos:**
![image](https://github.com/immich-app/immich/assets/28405293/8d823d6d-15c0-4304-9e8e-45dd975a0614)

google photos' logic is basically:
- side pane showing → activity status: no text + bottom row
- side pane hidden + <600px width → activity status: no text + third row ish
- side pane hidden + >600px width → activity status: yes text + bottom row

Reply:
On google photos, this overlap is annoying, but let's say they fixed it by raising the activity status to third row as they do on narrow width viewport. Then the text is still unobtrusive, but only because it fades out after a few seconds, which immich doesn't currently do (is probably fairly important to work on for release?).

![image](https://github.com/immich-app/immich/assets/28405293/db5c9aad-dd57-4054-a310-b3982da829c4)